src/web/app/pages-static/request-page/instructor-request-form/instructor-request-form.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { InstructorRequestFormData } from './InstructorRequestFormData';
+
+// eslint-disable-next-line
+const URL_REGEX = /(https?:\/\/)?(www\.)[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,4}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)|(https?:\/\/)?(www\.)?(?!ww)[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,4}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/;
```

Comment:
Amazing regex lol 😁

Anyway, we may want to move this to the `FieldValidator` and `ApiConst` in the back-end, and just import this from `src/web/types/form-validator.ts` (you can take a look, there are some existing examples in those files).

Though for now, I guess we can probably just leave this here; we can move it in the PR that will integrate it with the backend.

Reply:
Could we add a small comment explaining what this regex is for? Just for better readability